### PR TITLE
Admin statistics/ docs

### PR DESCRIPTION
**Pull request by deepsidhu85**
## Description of changes
What did you change in this pull request?  Provide a description of files changed, user interactions changed, etc.  Include how to test your changes.

Updated user documentation for admin statistics page.

## Related issue
Link to the GitHub issue this pull request addresses using the `#issuenum` format.  If it completes an issue, use `Fixes #issuenum` to automatically close the issue.

## Checklist
Things for the developer to confirm they've done before the PR should be accepted:

~* [ ] CHANGELOG.md (and UPGRADING.md if necessary) updated with information for new change.~
~* [ ] Tests added (or description of how to test) for any new features.~
* [X] User documentation updated for UI or technical changes.

### CODE REVIEW

File: doc/user/administrator/index.md [hunk 1]
```diff
@@ -32,6 +32,19 @@ The panel has a top navbar with a "Return" button to return to the main applicat
...
+Viewing Statistics
+==================
+
+When the Administrative Panel is accessed you will be taken to the landing page which is the Statistics Page.
+
+![Admin panel statistics link.](images/admin-panel-stats.png)
+
+This page displays usage statistics for the number of analyses ran, projects created, samples created, users created, and users logged in within the default time period which is set to 7 days.
+
+Clicking on any of the graphs will take you to another page for you to see advanced stats where you can select different time periods as well as, different chart types.
```
Comment (joshsadam) [inline doc/user/administrator/index.md:44]: This is awkwardly worded.  Try `Clicking on a graph will take you to the full details of the graph.  There you can set different time periods to explore the data and select different chart types.`  or something like that.

**Review by joshsadam — CHANGES_REQUESTED** [review]
Looks good, just one minor wording change please.

File: doc/user/administrator/index.md (hunk 1 above)
Comment (deepsidhu85) [reply doc/user/administrator/index.md]: Fixed in 528b7e7b961704290515e51f21b4ff2892ae2052. Thanks

**Review by joshsadam — APPROVED** [review]
Outstanding

**Pull request merged**